packages/carbon-web-components/src/components/ui-shell/side-nav.ts

```diff
@@ -128,10 +129,16 @@ class BXSideNav extends HostListenerMixin(LitElement) {
   expanded = false;
 
   /**
-   * Usage mode of the side nav.
+   * If `true` will style the side nav to sit below the header
    */
-  @property({ reflect: true, attribute: 'usage-mode' })
-  usageMode = SIDE_NAV_USAGE_MODE.REGULAR;
+  @property({ type: String, reflect: true, attribute: 'is-child-of-header' })
+  isChildOfHeader = 'true';
+
+  /**
+   * Specify if the side-nav will be persistent above the lg breakpoint
+   */
+  @property({ type: String, reflect: true, attribute: 'is-persistent' })
+  isPersistent = 'true';
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true, attribute: 'is-persistent' })
  isPersistent = true;
```
Same with this one I think?